restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -136,16 +137,22 @@ protected void configure() {
                 bind(environment.getObjectMapper()).to(ObjectMapper.class);
               }
             });
+
```

Comment:
Just reordered endpoints by grouping, then alphabetic, no changes to what is registered.